pydicom/tests/test_uid.py

```diff
@@ -61,7 +61,8 @@ def test_equality(self):
         assert not self.uid == 'Explicit VR Little Endian'
         # Issue 96
         assert not self.uid == 3
-        assert not self.uid is None
+        is_none = self.uid is None
+        assert not is_none
 
```

Comment:
On a second thought, this does not test the == operator, but `is`, which cannot be overwritten, so I think this test is just unneeded as it is.

Reply:
the right syntax is `assert self.uid is not None`, but I agree, that you are not testing the equality overload.

Reply:
Ok, I removed the test. It is indeed not needed here.